### PR TITLE
Add version command and flag

### LINKED ISSUE #33: Add top level version command, (or --version, whichever is preferred)
https://github.com/spf13/cobra#version-flag

### PR DESCRIPTION
**Pull request by maxine-krebs**
resolves #33

adds nox version and nox --version that prints out the current version.
The version is set in the makefile during the build phase and reads the
current git tag.

Also some import housekeeping since I changed my github username because
I am fickle

### CODE REVIEW

File: cmd/nox/Makefile [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 GOINSTALL=$(GOCMD) install
-GOBUILD=$(GOCMD) build
+GOBUILD=$(GOCMD) build -ldflags "-X main.Version=$(VERSION)"
 GOCLEAN=$(GOCMD) clean
 GORUN =$(GOCMD) run main.go
 GOTIDY=$(GOCMD) mod tidy
 BINARY_NAME=nox
 RELEASECMD=goreleaser release
 
+
```
Comment (maxine-krebs) [inline cmd/nox/Makefile:11]: tina groan
```suggestion
```

File: cmd/nox/version.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+)
+
+func init() {
+	rootCmd.AddCommand(versionCmd)
+}
+
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Print the version number of Hugo",
+	Long:  `All software has versions. This is Hugo's`,
```
Comment (maxine-krebs) [inline cmd/nox/version.go:16]: yikes copy 🍝 
```suggestion
	Short: "the current nox version",
	Long:  `the current git tag for nox releases`,
```

**Comment by maxine-krebs** [comment]
Proof:

```
➜  nox (mxk/version-command|✔ ) make build
go build -ldflags "-X main.Version=$(git describe --tags --abbrev=0)" -o nox -v
➜  nox (mxk/version-command|✔ ) ./nox --version
v0.1.4
➜  nox (mxk/version-command|✔ ) ./nox version
v0.1.4
➜  nox (mxk/version-command|✔ )
```

**Comment by maxine-krebs** [comment]
sorry for the noise and revert commits, I started down a path and then realized it really needed to be its own branch. im sure there is some fancy git way to more cleanly get rid of those commits but:

git

**Review by awseward — APPROVED** [review]

**Pull request merged**